packages/tailwindcss/src/index.ts

```diff
@@ -308,6 +340,16 @@ async function parseCss(css: string, { loadPlugin = throwOnPlugin }: CompileOpti
     return WalkAction.Skip
   })
 
+  for (let file of resolvedConfig.content.files) {
+    if ('raw' in file) {
+      throw new Error(
+        `Error in the config file/plugin/preset. The \`content\` key contains a \`raw\` entry:\n\n${JSON.stringify(file, null, 2)}\n\nThis feature is not currently supported.`,
+      )
+    }
+
+    globs.push(file.pattern)
+  }
```

Comment:
Note that this currently throws away `file.base`. We'll be fixing this up with another change briefly.